aliquotmaf/annotators/gnomad.py

```diff
@@ -0,0 +1,164 @@
+"""
+Implements the gnomAD annotation.
+"""
+
+from __future__ import absolute_import
+
+import os
+from itertools import repeat
+from sys import path
+
+import pandas as pd
+
+from aliquotmaf.converters.builder import get_builder
+
+from .annotator import Annotator
+
+GNOMAD_SOURCE_COLUMNS = (
+    'AF_non_cancer_eas',
+    'AF_non_cancer_afr',
+    'AF_non_cancer_ami',
+    'AF_non_cancer_mid',
+    'AF_non_cancer_sas',
+    'AF_non_cancer_nfe',
+    'AF_non_cancer',
+    'AF_non_cancer_amr',
+    'AF_non_cancer_oth',
+    'AF_non_cancer_asj',
+    'AF_non_cancer_fin',
+    'MAX_AF_non_cancer_adj',
+    'POP_MAX_non_cancer_adj',
+)
+
+GNOMAD_MAF_COLUMNS = (
+    "gnomAD_non_cancer_EAS_AF",
+    "gnomAD_non_cancer_AFR_AF",
+    "gnomAD_non_cancer_AMI_AF",
+    "gnomAD_non_cancer_MID_AF",
+    "gnomAD_non_cancer_SAS_AF",
+    "gnomAD_non_cancer_NFE_AF",
+    "gnomAD_non_cancer_AF",
+    "gnomAD_non_cancer_AMR_AF",
+    "gnomAD_non_cancer_OTH_AF",
+    "gnomAD_non_cancer_ASJ_AF",
+    "gnomAD_non_cancer_FIN_AF",
+    "gnomAD_non_cancer_MAX_AF_adj",
+    "gnomAD_non_cancer_MAX_AF_POPS_adj",
+)
+
+GNOMAD_SRC_TO_MAF = dict(zip(GNOMAD_SOURCE_COLUMNS, GNOMAD_MAF_COLUMNS))
```

Comment:
Because this is a mapping, I think it's safer to create the dict directly (and it can be an OrderedDict if order matters):

```python
GNOMAD_SRC_TO_MAF = OrderedDict(
    AF_non_cancer_eas="gnomAD_non_cancer_EAS_AF",
   ...
)
```
Tedious, but ensures that reordering either list doesn't cause bugs.

Reply:
It would also be possible to make these lists Enums, since at least one of these values is used in a separate module (new Filter)

Reply:
:+1: